code/game/machinery/computer/arcade.dm

```diff
@@ -1027,3 +1027,285 @@
 #undef ORION_TRAIL_COLLISION
 #undef ORION_TRAIL_SPACEPORT
 #undef ORION_TRAIL_BLACKHOLE
+
+#define PROB_CANDIDATE_ERRORS 8.3
+
+#define PROB_UNIQUE_CANDIDATE 2
+#define UNIQUE_STEVE 1
+#define UNIQUE_MIME 2
+#define UNIQUE_CEO_CHILD 3
+#define UNIQUE_VIGILANTE 4
+
+// Defines for the game screens
+#define RECRUITER_STATUS_START 0
+#define RECRUITER_STATUS_INSTRUCTIONS 1
+#define RECRUITER_STATUS_NORMAL 2
+#define RECRUITER_STATUS_GAMEOVER 3
+
+/obj/machinery/computer/arcade/recruiter
+	name = "NT Recruiter Simulator"
+	desc = "Weed out the good from bad employees and build the perfect manifest to work aboard the station."
+	icon_state = "arcade_recruiter"
+	icon_screen = "nanotrasen"
+	circuit = /obj/item/circuitboard/arcade/recruiter
+	var/candidate_name
+	var/candidate_gender
+	var/age
+	var/datum/species/cand_species
+	var/planet_of_origin
+	var/job_requested
+	var/employment_records
+	/// Current "turn" of the game
+	var/curriculums
+	/// Which unique candidate is he?
+	var/unique_candidate
+
+	var/list/planets = list("Earth", "Mars", "Luna", "Jargon 4", "New Cannan", "Mauna-B", "Ahdomai", "Moghes",
+							"Qerrballak", "Xarxis 5", "Hoorlm", "Aurum", "Boron 2", "Kelune", "Dalstadt")
+	/// Planets with either mispellings or ones that cannot support life
+	var/list/incorrect_planets = list("Eath", "Marks", "Lunao", "Jabon 4", "Old Cannan", "Mauna-P",
+									"Daohmai", "Gomhes", "Zrerrballak", "Xarqis", "Soorlm", "Urum", "Baron 1", "Kelunte", "Daltedt")
+
+	var/list/jobs = list("Assistant", "Clown", "Chef", "Janitor", "Bartender", "Barber", "Botanist", "Explorer", "Quartermaster",
+						"Station Engineer", "Atmospheric Technician", "Medical Doctor", "Coroner","Geneticist", "Chaplain", "Librarian",
```

Comment:
```suggestion
						"Station Engineer", "Atmospheric Technician", "Medical Doctor", "Coroner", "Geneticist", "Chaplain", "Librarian",
```

Reply:
I'm also curious if you might be able to pull this list from the existing jobs list/alt titles in some way

Reply:
Might be possible but i want to limit it to non command jobs and to more easily misspell them over ```incorrect_jobs```